binding/lua/init.lua

```diff
@@ -28,8 +28,15 @@ libmv = ffi.load(libmv_path, 'true')
 mv.ArrayTableHandler = require('multiverso.ArrayTableHandler')
 mv.MatrixTableHandler = require('multiverso.MatrixTableHandler')
 
-function mv.init(args)
-    args = args or {}
+function mv.init(sync)
+    if sync == nil then
```

Comment:
I think the three choices make no difference here.   I choose this one because other  two will convert any false value to `False`. Though it doesn't matter. It make me feel strange.
I've modified it to the shorter one now